ldaptor/protocols/ldap/ldapclient.py

```diff
@@ -109,10 +109,11 @@ def send_multiResponse(self, op, handler, *args, **kwargs):
 
         If `handler` is provided, it will receive a LDAP response as
         its first argument. The Deferred returned by this function will
-        never fire.
+        fire with a result of `None` when the handler returns True to indicate
+        the final response has been handled.
 
         If `handler` is not provided, the Deferred returned by this
-        function will fire with the final LDAP response.
+        function will fire with the first LDAP response.
```

Comment:
This comment was wrong, as is evidenced by the updated `test_send_multiResponse_with_handler` test